components/x-content-block/src/Timestamp.jsx

```diff
@@ -0,0 +1,45 @@
+import { h } from '@financial-times/x-engine';
+import styles from './ContentBlock.scss';
+
+// this function extracts time in HH:mm format from a given date
```

Comment:
Does `o-date` or any other existing package do this? Seems like it must be a common use case for elsewhere on FT.com.

Reply:
Now using `o-date` for rendering the exact date.

Reply:
Can the code that calculates the time ll.5-23 not use one of the `o-date` methods listed under the [JavaScript](https://github.com/Financial-Times/o-date#javascript) header?

It seems like what you are trying to achieve must be a common use case and I would be surprised if it was not achievable with one of these methods.

Reply:
I avoid using Origami Javascript in the x-dash component because of this comment here: https://github.com/Financial-Times/x-dash/issues/446#issuecomment-623949600

Reply:
Ah, interesting.